app/views/shared/_header.html.erb

```diff
@@ -1,20 +1,16 @@
-<nav class="navbar navbar-dark bg-primary">
+<nav class="l-header wrap-full">
   <button class="navbar-toggler hidden-sm-up" type="button" data-toggle="collapse" data-target="#exCollapsingNavbar2">
     &#9776;
   </button>
-  <div class="collapse navbar-toggleable-xs" id="exCollapsingNavbar2">
```

Comment:
Be careful when changing partials, they might be used in more than one place. This in particular is used both on the home and on the admin, and it seems to me that this is now broken on both of them.